css/css-position/sticky/position-sticky-stacking-context-002.html

```diff
@@ -25,14 +25,25 @@
 
   div#sticky
     {
-      background-color: green;
+      background: linear-gradient(to bottom, green 50%, red 50%);
```

Comment:
I suggest to use 51% green 49% red. This will increase the overlap by 1px, which should avoid any issues with anti-aliasing at the boundary. (I'm not sure if it's a problem, but maybe it's a problem.)

Reply:
It was not causing a problem in my opinion. I used 51% green 49% red anyway.